pyproject.toml

```
@@ -106,7 +112,7 @@ imap_xtce = 'imap_processing.ccsds.excel_to_xtce:main'
 ignore-words-list = "livetime"
 
 [tool.poetry-dynamic-versioning]
-enable = true
+enable = false
```

This should be reverted I think. When I checked this out I got a 0.0.0 version identifier installed. But enabling the dynamic versioning I then got the new hashed version.
```suggestion
enable = true
```

---

Oh yeah, definitely. This PR is a hotfix type suggestion and I was having some minor issues with the dynamic versioning when testing it. But, I got a reply from the plugin maintainer about this which makes using it a lot simpler, so we can do it the new way instead https://github.com/mtkennerly/poetry-dynamic-versioning/issues/197#issuecomment-2576291495